internal/cluster/transition.go

```diff
@@ -364,7 +364,8 @@ func (th *transitionHandler) WithAMSSubscriptions(sw stateswitch.StateSwitch, ar
 	if !ok {
 		return false, errors.New("WithAMSSubscriptions invalid argument")
 	}
-	if params.ocmClient != nil && params.ocmClient.Config.WithAMSSubscriptions && !sCluster.cluster.IsAmsSubscriptionConsoleUrlSet {
+	if params.ocmClient != nil && params.ocmClient.Config.WithAMSSubscriptions &&
+		!sCluster.cluster.IsAmsSubscriptionConsoleUrlSet && params.clusterAPI.IsOperatorAvailable(sCluster.cluster, operators.OperatorConsole.Name) {
```

Comment:
In that case, you can remove the `IsOperatorAvailable` condition in `PostUpdateFinalizingAMSConsoleUrl`

Reply:
I though I forgot in in the original PR so I added it here, but apparently I didn't :) 
I will remove one of them.